src/patterns/addresses/index.md

```diff
@@ -71,15 +71,6 @@ Make sure errors follow the guidance in [error message](/components/error-messag
 
 ## Address lookup
 
```

Comment:
```suggestion

An address lookup helps users find an address from a postcode and, optionally, a street number.

```

Reply:
@calvin-lau-sig7 - are we saying that postcode is always mandatory and street number always optional? And that the second field should always be labelled "street number" rather than for example "Address line 1" or something else?

How about:
> An address lookup helps users find a full address from partial information such as a postcode.

Reply:
Your suggestion looks great to commit, thank you! No, I'd just grabbed that sentence to help set context for the section.

Reply:
> Your suggestion looks great if you'd like to commit it, thank you! 
OK, how do I do that?

Reply:
```suggestion

An address lookup helps users find a full address from partial information such as a postcode.

```